monkey/tests/unit_tests/monkey_island/cc/island_event_handlers/test_agent_heartbeat_monitor.py

```diff
@@ -70,17 +79,34 @@ def test_multiple_agents(agent_heartbeat_handler, in_memory_agent_repository):
     assert agent_1.stop_time == datetime.fromtimestamp(110, tz=pytz.UTC)
     assert agent_2.stop_time == datetime.fromtimestamp(200, tz=pytz.UTC)
     assert agent_3.stop_time == agent_3.start_time
+    assert mock_island_event_queue.publish.call_count == 3
+    assert mock_island_event_queue.publish.called_with(
+        IslandEventTopic.AGENT_TIMED_OUT, agent_id=AGENT_ID_1
+    )
```

Comment:
I don't think "called_with()" actually exists? Did you red-green-refactor? That is, did you write the test, show that it failed, and then fix the code to make the test pass? If not, you may have a false positive here.

```suggestion
   mock_island_event_queue.publish.assert_called_with(
        IslandEventTopic.AGENT_TIMED_OUT, agent_id=AGENT_ID_1
    )
```